index.html

```diff
@@ -5,9 +5,17 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
+    <style>
+      *{
+         font-family: Arial, Helvetica, sans-serif !important;
+      }
+      body{
+         font-family: Arial, Helvetica, sans-serif !important;
+      }
+    </style>
   </head>
-  <body>
-    <div id="root"></div>
+  <body style="background-color:#1E1F21; font-family: Arial, Helvetica, sans-serif !important;" >
+    <div id="root" style= "min-height: 100vh;"></div>
```

Comment:
don't set height to the body, we don't need horizontal scroll!

```suggestion
  <body style="background-color:#1E1F21" >
    <div id="root"></div>
```